src/components/organisms/NotePageToolbar.tsx

```diff
@@ -49,6 +50,7 @@ import {
 } from '../../lib/electronOnly'
 import NotePageToolbarFolderHeader from '../molecules/NotePageToolbarFolderHeader'
 import path from 'path'
+import pathParse from 'path-parse'
```

Comment:
We don't need this. path is having `path.parse`. Please check the dev dependencies

Reply:
Please ignore this. I'll investigate more